config/action.d/badips.py

```diff
@@ -80,13 +82,14 @@ class BadIPsAction(ActionBase):
 	"""
 
 	_badips = "http://www.badips.com"
-	_Request = partial(
-		Request, headers={'User-Agent': "Fail2Ban %s" % f2bVersion})
+	def _Request(self, url, **argv):
+		return Request(url, headers={'User-Agent': self.agent}, **argv)
```

Comment:
Well, I saw that was used quasi as lambda (older py-versions?), so I've replaced it with def (to clarify the code and to remove an unnecessary module import);
What should be wrong with indentation? I see nothing here... Both (new and old version) set the member `_Request` in scope of class.